tests/contrib/anthropic/cassettes/anthropic_hello_world.yaml

```diff
@@ -0,0 +1,86 @@
+interactions:
```

Comment:
Let's reuse an existing cassette that covers a completion case instead of making a new one for each new test.

Reply:
done

Reply:
Looks like we don't need this cassette anymore.